src/VerifyWindowsHelloController.js

```diff
@@ -57,6 +57,10 @@ function (Okta, FormController, FormType, webauthn, Spinner, FooterSignout, Wind
                 signatureData: assertion.signature
               });
             })
+            .then(function (data) {
+              model.trigger('signIn');
```

Comment:
not sure I follow what is going on exactly, since this seems to me is not needed due to the ```trigger('sync')``` in line 53, which will show the spinner
And looking a little bit more at this, we should show the spinner and hide the button before we do the call to ```weauthn.getAssertion(``` in line 48, right? 
And then hide the spinner and show the button if the user cancels or there was an error.